include/osquery/transport.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Can you move this to `./osquery/remote/transports/tls.h` or place the define into another existing header within the codebase?

Also, what do you think about differentiating headers that define and construct values with a `.hpp`?